entries/c/cpp.edu.json

```diff
@@ -0,0 +1,27 @@
+{
+    "Cal Poly Pomona": {
+        "domain": "cpp.edu",
+        "url": "https://www.cpp.edu",
+        "img": "cpp.edu.png",
```

Comment:
SVG icons are preferred as they can be resized without losing details.
I suggest you use this image instead:
```SVG
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 107 107"><path fill="#5f7a99" d="m0 53.5 7.2 18.7 8.5 19.2V69.2zm78.2-27.7-9-10.1-15.8.4 24.4 10.1zm-9-10.1 9 10.1 10.7-10.2z"/><path fill="#97a7bc" d="M15.7 15.7 7.1 34.9 0 53.5l15.7-15.6zM53.5 0 34.3 7.4l-18.6 8.3h22.2zM16 53.5H0l15.7 15.7.3-15.7z"/><path fill="#00416b" d="M53.5 107 37.9 91.4H15.7zm-.2-16.2.2 16.2 15.7-15.6-15.9-.6zm-37.6.6L27 79.7 15.7 69.2zm53.5 0 18.9.2-9.8-10.4zm-15.9-.6L27 79.7l10.9 11.7z"/><path fill="#ffd100" d="m91.4 37.9-.6 15.6H107z"/><path fill="#ffb600" d="M91.4 69.2 107 53.5H90.8z"/><path fill="#385e82" d="M15.7 37.9 0 53.5h16zm22.2 53.5L53.5 107l-.2-16.2zm0-75.7 15.5.4.1-16.1zM27 27 15.7 15.7v22.2z"/><path fill="#597595" d="m53.5 0-.1 16.1 15.8-.4zM16 53.5 27 27 15.7 37.9zm-.3 15.7L27 79.7l-6.1-13.4L16 53.5z"/><path fill="#517091" d="M37.9 91.4 27 79.7 15.7 91.4z"/><path fill="#456789" d="m27 27 26.4-10.9-15.5-.4z"/><path fill="#798fa9" d="M37.9 15.7H15.7L27 27zm51-.1.6-.7L73.1 8 53.5 0l15.7 15.7z"/><path fill="#78c366" d="m80.7 29.1 10.1 24.4.6-15.6-10.1-9.3z"/><path fill="#3b6084" d="m88.1 91.6-18.9-.2L53.5 107l35.3-14.7z"/><path fill="#0e426a" d="m92.3 18.1-.9.8v19L107 53.5zm0 70.9L107 53.5 91.4 69.2l.1 19.1z"/><path fill="#00833e" d="M91.4 37.9v-19l-10.1 9.7zm0 31.3-9.9 8.5 10 10.6z"/><path fill="#6d85a2" d="m78.3 81.2-.5-.5-11.4 4.5-13.1 5.6 15.9.6z"/><path fill="#6abf4b" d="m81.5 77.7 9.9-8.5-.6-15.7L81 77.2z"/></svg>
```
![](https://d.tempfiles.download/D33F98C108834C/BYNRPqNOigGSPee2fn)

Reply:
I'll fix it in a bit, thanks!